src/Diagram/index.js

```diff
@@ -13,16 +16,18 @@ import { Element } from 'react-scroll'
 import Slider from 'react-slick'
 
 const LearnMore = ({ href }) => (
-  <LearnMoreArea href={href}>
-    <a href={href}>
-      <span>Learn&nbsp;more</span>
-      <img
-        src="/static/img/learn_more_arrow.svg"
-        width={18}
-        height={18}
-        alt=""
-      />
-    </a>
+  <LearnMoreArea>
+    <NextLink href={href}>
```

Comment:
Does NextLink work without js?

Reply:
I see it does but without js the nav bar doesn't expand. Unrelated to this PR I know, but should we care about this? Cc @shcheklein

Reply:
Working w/o JS is out of scope for this. The good question is if we need to generate a sitemap or something for the case when we don't have a link to some page inside and search engines won't be able to find it?

Reply:
Right, engined won't be able to crawl if the nav element doesn't SSR. A sitemap sounds like a good solution too but maybe it's easy to pre-render nav? Could open a separate `engine` issue for this.

Reply:
Yep, let's open and see what @iAdramelk has to say about this.

Reply:
OK, moving to #856.